api/prisma/migrations/20230320073247_init/migration.sql

```diff
@@ -30,7 +30,6 @@ CREATE TABLE "Adapter" (
 CREATE TABLE "Aggregator" (
     "id" BIGSERIAL NOT NULL,
     "aggregatorHash" TEXT NOT NULL,
-    "active" BOOLEAN NOT NULL DEFAULT false,
```

Comment:
@bayram98 We need to have this field. We just don't want to have it inside of the aggregator files.